docs/snippets/common/omponent-story-auto-title.csf3-story.js.mdx

```diff
@@ -0,0 +1,16 @@
+```js
+// src/components/MyComponent.stories.js|jsx
+
+import { MyComponent } from './MyComponent';
+
+/**
+ * Story written in CSF 3.0 with auto title generation
+ * See https://storybook.js.org/docs/7.0/react/api/csf
+ * to learn more about it.
+ */
+export default { component: MyComponent };
+
+export const Default = {
+  args: { something: 'Something else' },
```

Comment:
This might be silly, and feel free to disregard this request, but I feel like having example code like this gives a bit more credibility to the docs:

```suggestion
  args: { message: 'Hello world!' },
```